nautobot/project-static/css/dark.css

```diff
@@ -165,4 +165,27 @@ html[data-theme="dark"] .filter-container .filter-selection {
 html[data-theme="dark"] .filter-container .filter-selection .filter-selection-choice {
   background-color: rgb(185, 185, 185);                 /* Background color of dynamic filters selected across most pages, inner selection box */
   border: 1px solid rgb(170, 170, 170);                 /* Border color of dynamic filters selected across most pages, inner selection box */
-}
\ No newline at end of file
+}
+
+
+/* Buttons */
+html[data-theme="dark"] .btn-default {
+    background-color: rgba(0,0,0,0) !important;
```

Comment:
Instead of making the background colour transparent only in the dark theme, I propose making it transparent everywhere because the button background in the default/white theme is white and  in the dark/black theme is black, so making it transparent in one place should suffice.